### PR TITLE
adding in an example using eks for a public node

### PR DESCRIPTION
**Pull request by joshuafernandes**
Added a tutorial to run a besu node on AWS EKS

### CODE REVIEW

File: docs/public-networks/tutorials/kubernetes.md [hunk 1]
```diff
@@ -0,0 +1,276 @@
+---
+title: Deploy a Hyperledger Besu node with Kubernetes
+description: Deploying Hyperledger Besu with Kubernetes
+tags:
+  - public networks
+---
+
+# Deploy a Besu public node using Kubernetes
+
+You can use a cloud provider like AWS and Elastic Kubernetes Service to deploy a public node. For brevity we will assume you have an existing cluster setup and we add an extra nodegroup for your Besu pod
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:10]:
```suggestion
You can use a cloud provider like AWS and Elastic Kubernetes Service to deploy a public node. For brevity we will assume you have an existing cluster setup and we add an extra nodegroup for your Besu pod.
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 2]
```diff
@@ -0,0 +1,276 @@
...
+|Type|Protocol|Port Range|Destination|Description|
+|---|---|---|---|---|
+|Custom UDP|UDP|9000|0.0.0.0/0|CL client|
+|Custom TCP|TCP|9000|0.0.0.0/0|CL client|
+|Custom UDP|UDP|30303|0.0.0.0/0|EL client|
+|Custom TCP|TCP|30303|0.0.0.0/0|EL client|
+
+:::important
+
+The key here is to allow traffic on both TCP and UDP for the CL client and the EL client
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:34]:
```suggestion
The key here is to allow traffic on both TCP and UDP for the CL client and the EL client.
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 3]
```diff
@@ -0,0 +1,276 @@
...
+
+:::
+
+### Add a Nodegroup to your cluster
+
+We do this step so you can isolate your Besu node on a **public subnet** and seperate it from the other apps and nodegroups you may have running. If you are using something like
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:40]:
```suggestion
We do this step so you can isolate your Besu node on a **public subnet** and separate it from the other apps and nodegroups you may have running. If you are using something like
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 4]
```diff
@@ -0,0 +1,276 @@
...
+      }
+    ...
+
+```
+
+### Install the EBS or EFS drivers 
+
+We recommend using EBS or NvME storage for you chain data. For most cases the [EBS drivers](https://docs.aws.amazon.com/eks/latest/userguide/ebs-csi.html) or
+[EFS drivers](https://docs.aws.amazon.com/eks/latest/userguide/efs-csi.html) are sufficient. However if you are using instance stores, please use the
+[Local Storage Static Provisioner](https://aws.amazon.com/blogs/containers/eks-persistent-volumes-for-instance-store/) instead
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:100]:
```suggestion
[Local Storage Static Provisioner](https://aws.amazon.com/blogs/containers/eks-persistent-volumes-for-instance-store/) instead.
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 5]
```diff
@@ -0,0 +1,276 @@
...
+
+
+### Pod setup
+
+Now that the infrastructure is setup, we will use `hostNetworking` to bind your pod to the host and use the host node's public IP for you Besu node.
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:105]:
```suggestion
Now that the infrastructure is setup, we will use `hostNetworking` to bind your pod to the host and use the host node's public IP for your Besu node.
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 6]
```diff
@@ -0,0 +1,276 @@
...
+        value: "true"
+        effect: NoSchedule
+      - key: ethereum
+        value: "true"
+        effect: NoExecute
+```
+
+If you use something like Terraform, use something like this for your new nodepool
+
+```yaml
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:70]: think this needs a newline after it

File: docs/public-networks/tutorials/kubernetes.md [hunk 7]
```diff
@@ -0,0 +1,276 @@
...
+    labels:
+      ....
+    spec:
+      hostNetwork: true
+      dnsPolicy: ClusterFirstWithHostNet
+      affinity: ....
+```
+
+The next step is to add an init container and a shared volume to store the public IP. The init container `init` runs and gets the public IP of the host via the 
+AWS metadata service and saves it to a local shared volume `besu-pip` (between the init container and the besu pod)
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:122]:
```suggestion
AWS metadata service and saves it to a local shared volume `besu-pip` (between the init container and the besu pod).
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 8]
```diff
@@ -0,0 +1,276 @@
...
+      - name: besu-data
+        persistentVolumeClaim:
+          claimName: besu-pvc
+      - name: teku-data
+        persistentVolumeClaim:
+          claimName: teku-pvc             
+
+```
+
+When you start Besu up in the pod, use the text file in `pip` as your `p2p-host` and that will allow traffic in and out as normal
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:179]:
```suggestion
When you start Besu up in the pod, use the text file in `pip` as your `p2p-host` and that will allow traffic in and out as normal.
```

File: docs/public-networks/tutorials/kubernetes.md [hunk 9]
```diff
@@ -0,0 +1,276 @@
...
+  - public networks
+---
+
+# Deploy a Besu public node using Kubernetes
+
+You can use a cloud provider like AWS and Elastic Kubernetes Service to deploy a public node. For brevity we will assume you have an existing cluster setup and we add an extra nodegroup for your Besu pod.
+
+### Create a Security group for discovery
+
+The first step in this process is to create a security group in your VPC that allows traffic from the anywhere on port 30303 and 9000 (or equivalent ports that you are using for discovery).
```
Comment (macfarla) [inline docs/public-networks/tutorials/kubernetes.md:14]:
```suggestion
The first step in this process is to create a security group in your VPC that allows traffic from anywhere on port 30303 and 9000 (or equivalent ports that you are using for discovery). 
```

**Comment by macfarla** [comment]
couple of comments. I can't verify the k8s content and may need a docs expert to help fix the linting issues

preview https://besu-docs-git-k8s-public-networks-hyperledger.vercel.app/development/public-networks/tutorials/kubernetes

**Comment by joshuafernandes** [comment]
> couple of comments. I can't verify the k8s content and may need a docs expert to help fix the linting issues
> 
> preview https://besu-docs-git-k8s-public-networks-hyperledger.vercel.app/development/public-networks/tutorials/kubernetes

I'm going do a test of it early next week and then merge. The tf pieces Im sure of, the eksctl I'll confirm

**Review by alexandratran — APPROVED** [review]
Edited tutorial for clarity/style. LGTM.

**Pull request merged**